debian/control

```diff
@@ -0,0 +1,15 @@
+Source: bbb-plugin-h5p
+Section: web
+Priority: extra
+Maintainer: Anton Georgiev <anton.georgiev@blindsidenetworks.com>
+Build-Depends: debhelper (>= 13), nodejs (>= 18), unzip
+Standards-Version: 4.1.4
+Homepage: https://github.com/bigbluebutton/plugin-h5p
+
+Package: bbb-plugin-h5p
+Architecture: all
+Depends: ${misc:Depends}, nodejs
+Description: Share a webpage with all session participants
```

Comment:
Please write a description of the h5p plugin. I realize I missed this on the other plugins I pulled template over.